scripts/elysium.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+let fullName = "Lekan Adetunmbi";
```

Comment:
hey, @FElysian you can change 'fullName' variable to 'name' and don't forget to update the same on your pass variable for the output.